src/System.Reflection/tests/project.json

```diff
@@ -1,13 +1,16 @@
 {
   "dependencies": {
+    "System.AppContext": "4.0.0",
     "System.Collections": "4.0.10",
     "System.Diagnostics.Debug": "4.0.10",
     "System.Globalization": "4.0.10",
     "System.IO": "4.0.10",
+    "System.IO.FileSystem": "4.0.0.0",
     "System.Linq": "4.0.0",
-    "System.Reflection": "4.0.10",
+    "System.Reflection.Emit": "4.0.0",
```

Comment:
This + the System.Runtime.Loader dependency just caused these tests to not be runnable on .NET Native. We should consider splitting these out so we can still use these reflection tests on more of our platforms.